multiqc/utils/search_patterns.yaml

```diff
@@ -71,6 +71,9 @@ flexbar:
 gatk/varianteval:
     contents: '#:GATKTable:TiTvVariantEvaluator'
     shared: true
+gatk/base_recalibrator:
+    contents: '#:GATKTable:Arguments:Recalibration argument collection values used in this run'
```

Comment:
If you're happy with just `'#:GATKTable:Arguments:Recalibration'` being unique I'd probably go for that. I tend to go for the shortest unique string that I can, to future proof against tool authors changing wording in the future.